web/src/test/auto/integrated/cases/element_interfaces.js

```diff
@@ -413,6 +413,36 @@ if(typeof InterfaceTests == 'undefined') {
       String.kmwEnableSupplementaryPlane(false);
     }
 
+    InterfaceTests.Tests.getSelectedText = function(testObj) {
+      var Apple = InterfaceTests.Strings.Apple;
+      var pair = testObj.setupElement();
+
+      String.kmwEnableSupplementaryPlane(false);
+      testObj.resetWithText(pair, Apple.normal);
```

Comment:
`Apple.normal === '12345'`.  Not sure when that edit was made, but that threw me off for a little bit... so I thought it worth clarifying here.